supp2u/src/App.sass

```diff
@@ -87,3 +87,89 @@ form
 //input[type='integer']:focus,
 //textarea:focus + label
 //  display: none
+
+
+//SEARCH--------------------------------------------------------------------------------------------
+* 
+  box-sizing: border-box
+
+
+.movies 
```

Comment:
This looks good Laryna, my only suggestion would switch movies for restaurants or businesses.